substrate/client/service/src/builder.rs

```diff
@@ -529,6 +537,7 @@ where
 			config.blocks_pruning,
 			backend.clone(),
 			&*rpc_builder,
+			rpc_v2_metrics.clone(),
```

Comment:
Do we need the clone?

Reply:
Yep, since `gen_rpc_module` is a closure that get's called twice (once for normal rpc initialization and once for the inmemory initialization) 🙏